cocos/core/animation/playable.ts

```diff
@@ -23,6 +23,15 @@ export class Playable {
     get isPaused () {
         return this._isPaused;
     }
+
+    /**
+     * @en Is paused or stopped?
```

Comment:
Whether the current playable is stopped or paused

Reply:
Updated.